dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -336,6 +336,7 @@
 			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x1>;
 			st,mem2mem;
 			dma-requests = <7>;
+			dma-offset = <0>;
```

Comment:
DMAMUX is not present on all L4. It's only available on the L4R, L4S and L4P versions.
So I don't think this property should be added here (even if it doesn't hurt).

Reply:
Ah I added that becuase it didn't build on my L4R5 without it, didn't realize not all L4's need it, will take a look at where to place it so it only is present for SOC's that support it.

Reply:
I tend to prefer this: https://github.com/zephyrproject-rtos/zephyr/pull/30784/files#diff-cc2c619a82c50138c0684c40e7fa9e96a9416475c575dbf7ee64709fa7798a23.